lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -7089,7 +7090,7 @@ namespace Js
     {
 #if ENABLE_PROFILE_INFO
         // Switch off profiling is asmJsFunction
-        if (this->GetIsAsmJsFunction())
+        if (this->GetIsAsmJsFunction() || this->GetAsmJsModuleInfo())
```

Comment:
Is locking necessary here ? Even asmJsModuleInfo is set once by the AsmJs compiler, and remains read-only ?